conda_recipes/deadline/deadline-cloud.yaml

```diff
@@ -0,0 +1,12 @@
+condaPlatforms:
+  - platform: linux-64
+    defaultSubmit: true
+  - platform: linux-aarch64
```

Comment:
Thinking forward? I guess we do not have arm 64 (yet)

Reply:
I've been using this successfully with a Graviton customer-managed fleet.